core/scm.js

```diff
@@ -26,15 +27,15 @@ const SCHEMA_USER = Joi.object().keys({
 }).label('SCM User');
 
 const SCHEMA_REPO = Joi.object().keys({
-    id: Joi.string()
+    title: Joi.string()
```

Comment:
I feel like the name `title` and `subtitle` is a little confusing. Any reason why we want to name them like that?